source/developers/how-tos/ongoing-maintainers-tasks.rst

```diff
@@ -0,0 +1,107 @@
+Ongoing Maintainer's Tasks
+##########################
+
+..
+  Much of the TBD content in this document should be answered when we come
+  to consensus on https://openedx.atlassian.net/wiki/spaces/COMM/pages/3545726977/Maintainer+Outstanding+Questions+9+14+meeting
+
+
+This How-to will explain the ongoing tasks that Maintainers are expected to
+perform, once their repositories are up to Open edX project standards (see
+:doc:`maintain-a-repo`). These actions are performed **periodically** - some more
+frequently than others - and are done to ensure your repository is consistently
+up to date with the latest security fixes and the rest of the project's codebase.
+Additionally, maintainers have a responsibility to the community to develop
+transparently, respond to requests for help, and respond to proposals/PRs for
+changes to the codebase.
+
+To learn more about the Maintainer's Program, see `OEP-55`_.
+
+.. _OEP-55: https://open-edx-proposals.readthedocs.io/en/latest/processes/oep-0055-proc-project-maintainers.html
+
+
+Assumptions
+***********
+
+This How-to assumes the following:
+
+* You are a repository maintainer.
+
+* You have followed the instructions in :doc:`maintain-a-repo` to get your repo's
+  documentation and tooling in alignment with the standards of the project.
+
+GitHub Notifications
+********************
+
+GitHub provides a lot of individual control over how you receive notifications when you or teams you are on are mentioned in issues or pull-request on Github.com
+
+You can learn more about how to configure your notifications settings on `GitHub's notifications documentation`_ 
+
+.. note::
+
+   Expectations for how quickly you respond to issues or pull requests are detailed in the sections below.
+
+
+.. _GitHub's notifications documentation: https://docs.github.com/en/account-and-profile/managing-subscriptions-and-notifications-on-github/setting-up-notifications/about-notifications
+
+Responding to Issues
+********************
+
+Issues may come in the form of bug reports or feature requests, in any case, the issues should be **"triaged" within a week of receipt**.  This does not mean agreeing to the request in the issue or fixing it. But simply acknowledging it and scheduling time to respond to the requests in a timely manner.  Sometimes the issues will require further discussion but if you see issues that you are unlikely to resolve, it is better to provide that feedback quickly rather than leaving the issue open and unanswered.
```

Comment:
```suggestion
Issues may come in the form of bug reports or feature requests. In any case, the issues should be **"triaged" within a week of receipt**.  This does not mean agreeing to the request in the issue or fixing it, but simply acknowledging it and scheduling time to respond to the requests in a timely manner.  Sometimes the issues will require further discussion but if you see issues that you are unlikely to resolve, it is better to provide that feedback quickly rather than leaving the issue open and unanswered.
```